keras_cv/bounding_box/to_ragged_test.py

```diff
@@ -23,13 +23,27 @@ def test_converts_to_ragged(self):
                 [[[0, 0, 0, 0], [0, 0, 0, 0]], [[2, 3, 4, 5], [0, 1, 2, 3]]]
             ),
             "classes": tf.constant([[-1, -1], [-1, 1]]),
+            "confidence": tf.constant([[0.5, 0.7], [0.23, 0.12]]),
```

Comment:
How is there a confidence if no class is detected? Confused

Also, why is the one positive class label have the lowest confidence.

Reply:
Any box with a -1 class is considered padding.  These values could be anything - they’re irrelevant.

Reply:
(One case this could happen is if there is a high IOU between a box with a high confidence, and a box with EVEN higher confidence.  The high confidence box will be pruned, yielding a class ID of -1, but a high confidence.  Filtering out using to_ragged() removes these values which is best for UX